.vscode/settings.json

```diff
@@ -10,5 +10,12 @@
     ],
     "flake8.args": [
         "--max-line-length=120"
+    ],
+    "python.testing.unittestArgs": [
```

Comment:
What problem did adding these solve?

Reply:
Adding this setting is what made running/debugging the tests within VSCode work for me.
I could remove it or ignore the file/directory.

Reply:
I just tried it without those settings and running/debugging still works, so I restored the original version of this file.